prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -110,6 +123,116 @@ public async Task<ActionResult<Organization>> CreateOrganization(CreateOrganizat
             );
         }
 
+        // POST: api/Organizations/claim
+        /// <summary>
+        /// Claim an existing Organization.
+        /// </summary>
+        [HttpPost("claim", Name = nameof(ClaimOrganization))]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(ApiResultResponse<int>), StatusCodes.Status200OK)]
+
+        public async Task<ActionResult<Organization>> ClaimOrganization(ClaimOrganizationViewModel claimOrganization)
+        {
+            if (!await _partyService.PartyExistsAsync(claimOrganization.PartyId, PartyType.SigningAuthority))
+            {
+                return BadRequest("Could not claim an organization, the passed in SigningAuthority does not exist.");
+            }
+
+            var organization = await _organizationService.ClaimOrganizationAsync(claimOrganization.PartyId, claimOrganization.PEC, claimOrganization.ClaimDetail);
```

Comment:
You can pass in the View Model here rather than pulling out the parts. This reduces the method signature as well as reducing churn if we have to add/remove parameters from the VM.